openmetadata-spec/src/main/resources/json/schema/security/credentials/gcsCredentials.json

```diff
@@ -110,6 +128,19 @@
           "$ref": "#/definitions/GCSCredentialsPath"
         }
       ]
+    },
+    "gcpImpersonateServiceAccount": {
+      "title": "GCP Impersonate Service Account Configuration",
+      "description": "we enable the authenticated service account to impersonate another service account",
+      "oneOf": [
+        {
+          "$ref": "#/definitions/GCPImpersonateServiceAccountValues",
+          "default": {}
+        },
+        {
+          "type": "null"
+        }
+      ]
```

Comment:
I have added the new optional object in addition to the credentials values, because service account impersonation has to work based on the credentials.